uwuno/src/test/java/com/learning/uwuno/integration/tests/startGameTest.java

```diff
@@ -0,0 +1,93 @@
+package com.learning.uwuno.integration.tests;
+
+import com.learning.uwuno.integration.jsonUtil;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import static com.learning.uwuno.integration.constants.BASE_URL;
```

Comment:
Group similar packages with each other, easier to find stuff and know if we should use * if theres too many.

Reply:
Forgot to fix this